src/encord_active/lib/metrics/metric.py

```diff
@@ -52,23 +53,54 @@ class AnnotationType:
 AnnotationTypeUnion = Union[ObjectShape, ClassificationType]
 
 
-class MetricMetadata(TypedDict):
-    annotation_type: Optional[List[AnnotationTypeUnion]]
-    embedding_type: Optional[str]
-    data_type: DataType
-    long_description: str
-    metric_type: MetricType
-    needs_images: bool
-    short_description: str
-    title: str
+@dataclass
+class StatsMetadata:
```

Comment:
Only saw one usage of the min/max stats, so this is currently unused. Would like to know if the min/max/mean/threshold stats as part of metrics metadata are used anywhere.